packages/react-native/sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -22,8 +22,7 @@ end
 # package.json
 package = JSON.parse(File.read(File.join(react_native_path, "package.json")))
 # [macOS
-rn_version = package['version']
-version = findLatestVersionWithArtifact(rn_version) || rn_version
+version = findLatestVersionWithArtifact(package) || package['version']
```

Comment:
The name of the method is no longer correct? Can we rename to something like findMatchingHermesVersion or something? 

Also a not, can we keep the original upstream line of code we changed commented above so the diff is more clear (we don't have to, I'm good either way)

Reply:
The upstream version is just `package['version']`, so I think a tag mentioning the addition of `findMatchingHermesVersion` is cleaner.